core/dbt/events/functions.py

```diff
@@ -154,7 +154,8 @@ def event_to_dict(e: T_Event, msg_fn: Callable[[T_Event], str]) -> dict:
         'level': level,
         'data': Optional[Dict[str, Any]],
         'event_data_serialized': True,
-        'invocation_id': e.get_invocation_id()
+        'invocation_id': e.get_invocation_id(),
+        'thread_name': e.get_thread_name()
```

Comment:
(for later) Let's think about splitting `create_text_log_line` into `create_text_stdout_log_line` and `create_text_file_log_line`. I like how the thread name has been historically included in `logs/dbt.log`, but it feels like a lot of extra clutter on the CLI.

Reply:
#4354